articles/paper-prototyping.md

```diff
@@ -0,0 +1,54 @@
+issue: 52
+title: Why You Should Create a Paper Prototype - and How to Test it with Your Users
+---
+# {{page.title}}
+
+
+At Zühlke, we strive to work in a lean and agile way. We embrace the uncertainty by starting with a focus on exploring and learning. In interaction design, this can be achieved by creating, testing and iteratively refining a paper prototype. 
+
+## Why test a paper prototype?
+
+Some of the reasons are:
+
+1.	Getting the interaction design right should be your first priority: navigation, workflow, organisation of content and the terms used to denote domain elements. Simple black-and-white wireframes help you focus on these issues.
+2.	Action sequences or *scenarios* are paramount in interaction design. Creating a paper prototype makes you think about those sequences. Will the user understand how to perform the next step? And if so, how can they navigate back?
+3.	A paper prototype helps you get the most relevant feedback as early as possible. Do the users understand the navigation? Did you get their workflow right? Guess what: sometimes you didn't, but all you have to do to improve your design is change some wireframes.
+4.	Starting with a low-fidelity prototype means there’s a natural focus on the big picture. Once the interaction basics have been validated, there will be time to fine-tune all the details of your design.
+5.	Creating a paper prototype allows you to involve your stakeholders into the interaction design and evaluation process. Schedule design sessions with them. Have your stakeholders observe the user tests. You can even change the prototype on the fly while you are testing it!
+
+## Creating a paper prototype
+
+Here's the basic workflow:
+
+1.	Select the scenarios that you would like to test.
+2.	Create a set of wireframes, either directly on paper or using a tool like Balsamiq. Make sure to include all intermediate steps to your wireframes that a user would go through when performing the scenarios.
+3.	If one step involves adding a dialog on top of an existing page, create a wireframe "snippet" that just shows the dialog. This also works for other small changes on a screen.
+4.	The result is a stack of sketched or printed screens and a collection of snippets.
+
+![An paper prototype created using Balsamiq](./paper-prototyping/prototype-example.png)
+![An hand-sketched paper prototype](./paper-prototyping/paper-prototype-sketched.png)
+
+## Conducting the test
+
+Let "real" users of your system test the paper prototype. For a typical early-stage paper prototyping test, five representative users are enough to find the most important issues in your design. 
+
+Your goal during the test is to observe users performing the scenarios. It's very much like a regular usability test where you replace the computer with a stack of paper. To prepare the test, write an instruction sheet that asks the users to perform the tasks underlying the scenarios you used for creating the wireframes.
```

Comment:
Hilft es da noch spezifischer zu werden? 
Observe over the shoulder? Influencing the user with body language? 
Wahrscheinlich geht das zu weit...

Reply:
I thought about this but decided to leave it as it is. Trying to stick to the main points here, of course there's more to know about paper prototyping than what I wrote about...

Reply:
Thanks for the praise and the comments! I'm considering the article done for now.